Mathlib/Topology/Bases.lean

```diff
@@ -546,59 +559,59 @@ theorem IsSeparable.of_subtype (s : Set α) [SeparableSpace s] : IsSeparable s :
   simpa using isSeparable_range (continuous_subtype_val (p := (· ∈ s)))
 #align topological_space.is_separable_of_separable_space_subtype TopologicalSpace.IsSeparable.of_subtype
 
-@[deprecated] -- Since 2024/02/05
+@[deprecated IsSeparable.of_subtype] -- Since 2024/02/05
```

Comment:
Doesn't `@[deprecated] alias a := b` autogenerate correct hint?

Reply:
Ah, it does!